### ISSUE TITLE
Write an article clarifying what to expect from Does and vice versa

### ISSUE THREAD

**Issue by Robotorium** [DoESLiverpool/somebody-should#1961]
I run into people quite frequently still, who don't understand what the idea behind DoES is, and we also have some issues with people not understanding that a shared space includes shared responsibilities. I'm sure that's not fun for them, either, so I mentioned at the first of the recent Future Gazings that I thought we should have a specific document explaining all of this.

I had a go at this, it is here https://github.com/DoESLiverpool/somebody-should/wiki/What-to-Expect-from-DoES

Obviously feel free to edit as appropriate (would appreciate if someone could explain where the bins go because I don't know).

**Comment by huffeec** [comment]
Following the discussion at the community meeting can this page have a link to the how to github page
**[https://github.com/DoESLiverpool/somebody-should/blob/master/CONTRIBUTING.md](https://github.com/DoESLiverpool/somebody-should/blob/master/CONTRIBUTING.md)**

**Comment by Robotorium** [comment]
Done, cheers Chris!

Op zo 2 feb. 2025 13:51 schreef huffeec ***@***.***>:
